packages/data-stores/src/site/types.ts

```diff
@@ -463,6 +463,7 @@ export interface LaunchPadCheckListTasksStatuses {
 	site_launched?: boolean;
 	site_edited?: boolean;
 	video_uploaded?: boolean;
+	publish_first_course: boolean;
```

Comment:
Minor change, can you make this property optional? `publish_first_course?: boolean`

Reply:
Fixed in b656077c7c6d2d0b909ae5d2099b75930d19c640